.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Could you remove this file and update the `.gitignore` to `.vscode` outside of the files we have checked in there?